alacritty/src/event.rs

```diff
@@ -392,7 +394,7 @@ impl<N: Notify> Processor<N> {
                 message_buffer: &mut self.message_buffer,
                 display_update_pending: &mut display_update_pending,
                 window: &mut self.display.window,
-                font: &mut self.font,
+                font_size: self.font_size,
```

Comment:
You're copying the font size here and later assigning it to `self.font_size` again. Couldn't we just pass `&mut self.font_size`?

Reply:
Good point, I've updated this appropriately. Thanks for the quick review!